Jenkinsfile.d/core/package

```diff
@@ -83,7 +83,7 @@ pipeline {
     GPG_FILE                  = 'jenkins-release.gpg'
     GPG_PASSPHRASE            = credentials('release-gpg-passphrase-2023')
     PACKAGING_GIT_REPOSITORY  = 'git@github.com:jenkinsci/packaging.git'
-    PACKAGING_GIT_BRANCH      = 'master'
+    PACKAGING_GIT_BRANCH      = 'stable-2.414'
```

Comment:
Since the target branch of this pull request is the master branch, shouldn't this remain as "master" rather than "stable-2.414"?
```suggestion
    PACKAGING_GIT_BRANCH      = 'master'
```

Reply:
with the rebase, this change is no more. Thanks for the careful review!